<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MulrecDialog</class>
 <widget class="QDialog" name="MulrecDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debug Multiple Records</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QSpinBox" name="spinBox_FromSeg">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ToSeg">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_SegCount">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_To">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Of">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>of</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_DispData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>431</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Display data</string>
   </property>
   <widget class="QCheckBox" name="checkBox_OneSeg">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>One segment at a time</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_OnlySegs">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>For only the following segments</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_AllSegs">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>For all segments</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_Ok">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Cancel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <zorder>groupBox_DispData</zorder>
  <zorder>spinBox_FromSeg</zorder>
  <zorder>spinBox_ToSeg</zorder>
  <zorder>spinBox_SegCount</zorder>
  <zorder>label_To</zorder>
  <zorder>label_Of</zorder>
  <zorder>pushButton_Ok</zorder>
  <zorder>pushButton_Cancel</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_Ok</sender>
   <signal>clicked()</signal>
   <receiver>MulrecDialog</receiver>
   <slot>s_Ok()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>MulrecDialog</receiver>
   <slot>s_Cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_AllSegs</sender>
   <signal>toggled(bool)</signal>
   <receiver>MulrecDialog</receiver>
   <slot>s_ModeChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>399</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>s_Ok()</slot>
  <slot>s_Cancel()</slot>
  <slot>s_ModeChanged(bool)</slot>
 </slots>
</ui>
